roles/infinispan/tasks/install.yml

```diff
@@ -141,7 +146,7 @@
   ansible.builtin.unarchive:
     remote_src: yes
     src: "{{ archive }}"
-    dest: "{{ infinispan.installation_path | dirname | dirname }}"
+    dest: "{{ infinispan.installation_path | dirname }}"
```

Comment:
So the change in line 139 above makes sense, but this one on the `unarchive` needs to be reverted to `| dirname | dirname` (since the zipfile contains the directory inside). Tested on molecule, upstream and downstream

Reply:
Thanks for your testing efforts, reverted, rebased and force-pushed -- should be fine now